src/Search.js

```diff
@@ -41,27 +51,169 @@ export default function Search() {
     <Container>
       <Row>
         <Col>
+          <Container>
+            {showAlert && (
+              <Alert variant='info' onClose={() => setShowAlert(false)} dismissible>
+                <Alert.Heading>Usage</Alert.Heading>
+                <p>
+                  Click on the list in the right to add a card, use the selection above to filter
+                  the cards in the list
+                </p>
+              </Alert>
+            )}
+          </Container>
+
           <Form>
             <InputGroup className='my-3 mb-3 text-light'>
-              <Form.Control
-                style={{ borderColor: '#444' }}
-                className='bg-dark text-light'
-                onChange={(e) => setSearch(e.target.value)}
-                placeholder='Search'
-                value={search}
-              ></Form.Control>
               <Form.Select
                 style={{ borderColor: '#444' }}
                 className='bg-dark text-light'
                 value={quality}
                 onChange={(e) => setQuality(parseInt(e.target.value))}
+                isInvalid={quality == 0}
               >
+                <option disabled value={0}>
+                  Quality
+                </option>
                 {['Diamond', 'Gold', 'Shadow', 'Meteorite', 'Plain'].map((e, i) => (
```

Comment:
try to use better variable names. for another Developer `e` is nothing but a letter. But, for example

```jsx
const qualities = ['Diamond', 'Gold', 'Shadow', 'Meteorite', 'Plain']

{qualities .map((quality, i) => (
  <option key={i} value={1 + i}>
    {quality}
  </option>
))}
```

is more descriptive